falco.yaml

```diff
@@ -375,3 +375,49 @@ metadata_download:
   max_mb: 100
   chunk_wait_us: 1000
   watch_freq_sec: 1
+
+
+# [EXPERIMENTAL] additional_syscalls option !!! WARNING use with caution !!!
```

Comment:
@FedeDP more thoughts on the docs for the first iteration?

Reply:
Few more thoughts as I was prepping an internal build and update rules for an internal test deployment rollout:

- Need to add that it's also all resolved evttypes from macros not just the rules, just to be very clear here
- Even more clearly say that whatever is in the rules + macros and the additional_syscalls is all that is activated, nothing else when additional_syscalls option is choosen, in case the "state enforcement" phrasing is too obscure for the uninitiated 
- @Happy-Dude as I was testing it now also against the upstream rules, I noticed that it would have been really nice to have that "dry-run" option similar to `-V` that in addition to those global lists of syscalls also included a listing of the rules and macros associated with a particular syscalls -> let's make it happen for sure!